docs/install.md

```diff
@@ -1,6 +1,7 @@
 # Installation/Update
 
 ## Dependencies
+* 64bit OS (starting with News 16.0.0)
 * PHP >= 7.2
 * Nextcloud 20
 * libxml >= 2.7.8
```

Comment:
```suggestion
* libxml >= 2.7.8

```
otherwise it is
- libxml >= 2.7.8 You also need some PHP extensions:

Reply:
fixed